framework/src/new_operators/queue_batch.rs

```diff
@@ -0,0 +1,96 @@
+use super::{Batch, PacketError};
```

Comment:
We should probably have some module documentation here.

Reply:
@drunkirishcoder just sayin :).

Reply:
I will add more doc in the next PR where I'm expanding on the queue batch functionality with packet generation.

Reply:
good deal.